cmd/minikube/cmd/config/validations.go

```diff
@@ -50,6 +51,14 @@ func IsValidDiskSize(name string, disksize string) error {
 	return nil
 }
 
+func IsValidURL(name string, location string) error {
+	_, err := url.Parse(location)
```

Comment:
I'm not really sure the desired behavior, it just seems strange to do a check that seems to allow everything.